src/main/java/strategy/operation/FruitBalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package strategy.operation;
+
+import strategy.FruitStorageHandler;
+import strategy.StorageHandler;
+
+public class FruitBalanceOperation implements OperationHandler<String, Integer> {
+    private static StorageHandler<String, Integer> STORAGE_HANDLER = new FruitStorageHandler();
```

Comment:
same as were above
private final and injected via constructor
please recheck all operations